.sonarcloud.properties

```diff
@@ -1,3 +1,3 @@
 sonar.cpd.exclusions=src/foundation/Icon/IconsMap.tsx
-sonar.exclusions=src/foundation/Icon/IconsMap.tsx
+sonar.exclusions=src/foundation/Icon/IconsMap.tsx,src/components/Button/ButtonClasses.tsx
```

Comment:
Nice!!! This is exactly what I was looking for as we had a serious issue of duplication on that file. 🙌🏼